services/storage/tests/unit/test_handlers_simcore_s3.py

```diff
@@ -192,7 +192,7 @@ async def test_copy_folders_from_valid_project_with_one_large_file(
     create_simcore_file_id: Callable[[ProjectID, NodeID, str], SimcoreS3FileID],
     aiopg_engine: Engine,
     random_project_with_files: Callable[
-        ...,
+        [int, tuple[ByteSize], tuple[SHA256Str]],
```

Comment:
let me very pedantic and highlight that aws s3 etags are not only sha256 https://docs.aws.amazon.com/AmazonS3/latest/API/API_Object.html but also md5, sha1, etc. ;) :P

Reply:
yes but this does not mimick what AWS S3 can do but what we use. Now I'm not familiar with this part and probably @bisgaard-itis might have another take on it. Nevertheless I only fixed the typing.